app/models/text.rb

```diff
@@ -11,8 +11,18 @@ class Text < ApplicationRecord
     git: 2,
     ruby: 3,
     rails: 4,
-    php: 5
+    php: 5,
   }
-  
+
   RAILS_GENRE_LIST = %w[basic git ruby rails].freeze
+
+  PHP_GENRE_LIST = %w[php]
```

Comment:
定数の配列には `freeze` を入れておきましょう。

```suggestion
  PHP_GENRE_LIST = %w[php].freeze
```